DFC.App.DiscoverSkillsCareers.Services/Helpers/JobCategorySkillMappingHelper.cs

```diff
@@ -83,6 +83,11 @@ public static class JobCategorySkillMappingHelper
 
             var minProfileDistributionPercentage = 100 - maxProfileDistributionPercentage;
```

Comment:
What's the inferred type of this variable? make sure it's not int or use double instead of var?

Reply:
It is double so all okay - good spot tho